src/report/report.ts

```diff
@@ -1,6 +1,13 @@
 // (C) 2007-2019 GoodData Corporation
-import { ExecuteAFM } from "@gooddata/typings";
+import { AFM, ExecuteAFM } from "@gooddata/typings";
+import compact from "lodash/compact";
+import isEmpty from "lodash/isEmpty";
 import { ERROR_RESTRICTED_CODE, ERROR_RESTRICTED_MESSAGE } from "../constants/errors";
+import {
+    convertAbsoluteDateFilter,
+    convertRelativeDateFilter,
+} from "../DataLayer/converters/FilterConverter";
+import { convertFilter as convertAttributeFilter } from "../execution/execute-afm.convert";
```

Comment:
it's weird that it's in different file than other filterConverters. How about we fix it?

Reply:
It is weird. However, I would leave it as it is. DataLayer is removed in SDK8 and execute-afm conversion is handled differently as well. This would not help here much and would make the merging of changes to SDK8 much harder. Are you OK with that?

Reply:
sure